pkg/dispatcher/operator/workers/controller.go

```diff
@@ -101,6 +101,26 @@ func NewController(ctx context.Context, kubeClient kubernetes.Interface, podInfo
 	return controller
 }
 
+// Utility function
+func (c *Controller) deleteCalculation(pod *corev1.Pod, name string, key string) error {
```

Comment:
what is `name` and `key` here?

Reply:
Could you please explain what the key in syncHandler(key string) could be? I cant find a reference in the code. Its not clicking in for me how to use syncHandler to help me solve the problem.

Reply:
the `key` in syncHandler is a combination of the type of the namespace and name of the object. You can see in L132 that we split it.